src/main/java/user_send_message/MessageInputBoundary.java

```diff
@@ -20,15 +17,15 @@ public interface MessageInputBoundary {
 
     /**
      * Fetches all the messages in a specific chat ID
-     * @param chatID
+     * @param chatID ID of the chat
      * @return List of all messages in a chat
```

Comment:
forgot to update the javadoc. This method does not return a list of all messages.